wargames/bandit/bandit19.md

```diff
@@ -11,5 +11,5 @@ to log you out when you log in with SSH.
 
 Commands you may need to solve this level
 -----------------------------------------
-ssh, ls, cat
+scp, ssh, ls, cat
```

Comment:
Yeah. But it's also possible to solve it using scp only. I thought this challenge would be a good opportunity to learn about scp as well since, if I remember correctly, none of the other challenges talk about scp.